lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -50,6 +57,8 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
             self._delete_nodes(environment, log, qemu_conn)
 
     def _deploy_nodes(self, environment: Environment, log: Logger) -> None:
+        libvirt_events_thread.init()
```

Comment:
what's the life cycle of this thread? If it's longer than the node. Call it in `platform._initialize`. It calls multiple times when 
using combinator, but it can reduce calls than deploy.